### PR TITLE
My profile : infos + games

### PR DESCRIPTION
**Pull request by raphaeldieu**
Use the user_id of the user logged in and display his own onformations

changed identification by username to identification by id in users.js and games.js

### CODE REVIEW

File: public/application-index.js [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+    }
+  })
+};
+
+Profile.prototype.addGame = function(){
+  $.ajax({
+    method: 'POST',
+    url: '/sessions/raf/games',
+    data: {
+      name: 'Fallout',
```
Comment (f3r) [inline public/application-index.js:60]: why hardcoded?

**Comment by raphaeldieu** [comment]
Oups.. forgot to delete that one

**Comment by raphaeldieu** [comment]
It was just a test

File: public/application-main.js [hunk 2]
```diff
@@ -0,0 +1,175 @@
...
+  })
+};
+
+Session.prototype.displayMyGames = function(id){
+  $.ajax({
+    method:'GET',
+    url: '/users/'+ id,
+    success: function(response){
+      var html_games_user = '';
+      for (var i = 0; i < response.games.length; i++){
```
Comment (f3r) [inline public/application-main.js:65]: Cleaner way of doing this:

``` js
response.forEach(function(elem) {
  html_games_user += '<tr>';
  html_games_user +=    '<td>' + elem.name + '</td>';
  ...
```

File: public/application-main.js [hunk 3]
```diff
@@ -0,0 +1,175 @@
...
+};
+
+Database.prototype.displayDatabase = function() {
+  $.ajax({
+    context:this,
+    method:'GET',
+    url: '/games',
+    success: function(response){
+      var html_database = '';
+      for (var i = 0; i < response.length; i++){
```
Comment (f3r) [inline public/application-main.js:122]: same as before

**Pull request merged**